Clients/GUI/Pages/SignIn.razor

```diff
@@ -0,0 +1,66 @@
+﻿@page "/signin"
+@using Authentication
+@using ViewModels
+@using DTO
+@using Scripts
+@inject AuthStateProvider AuthenticationProvider
+@inject NavigationManager NavigationManager
+
+<h3 style="color: red; display: @(isHidden ? "none" : "block")"> Wrong login or password</h3>
+<div class="col-12">
+    <h3><b>Sign In</b></h3>
+    <hr />
+    <EditForm OnValidSubmit="@TrySignIn">
```

Comment:
Does it work properly? I thought we have to set the model explicitly like `Model="@userInput"` or sth like that

Reply:
I am not sure that binding will work without keyword _Model_